core/src/http_advertiser.cpp

```diff
@@ -0,0 +1,69 @@
+#include "include/http_advertiser.h"
+
+HTTPAdvertiser::HTTPAdvertiser()
+{
+    m_manager = new QNetworkAccessManager();
```

Comment:
This will probably not get deleted later.
May wanna make a destructor for HTTPAdvertiser, and deleteLater() this, I think.